tests/test_worker.py

```diff
@@ -1,11 +1,15 @@
 import asyncio
+from re import A
```

Comment:
What the heck is A?

Reply:
I have no freaking idea.
I really have to check my IDE settings. I keep importing weird things.